cmd/operator/certs.go

```diff
@@ -0,0 +1,59 @@
+package operator
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/scheme"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/certificates"
+	"github.com/pkg/errors"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+func runCertBootstrapper(cfg *rest.Config, namespace string) error {
+	bootstrapManager, err := createCertBootstapperManager(cfg, namespace)
```

Comment:
Is `bootstrap` correct? I always only saw `bootstrapper` as a name. Not that it is that important, just asking out of curiosity.

Reply:
it used to be called that, I tried to rename it so it doesn't conflict with the new bootstrapper

Reply:
renamed it. hopefully better